src/Networking/v2/Models/Loadbalancer.php

```diff
@@ -0,0 +1,151 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\Creatable;
+use OpenStack\Common\Resource\Deletable;
+use OpenStack\Common\Resource\OperatorResource;
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\Updateable;
+use OpenStack\Networking\v2\Api;
+
+/**
+ * Represents a Neutron v2 Loadbalancer
+ *
+ * @property Api $api
+ */
+class Loadbalancer extends OperatorResource implements Creatable, Retrievable, Updateable, Deletable
+{
+    /**
+     * @var string
+     */
+    public $name;
+
+    /**
+     * @var string
+     */
+    public $description;
+
+    /**
+     * @var boolean
+     */
+    public $adminStateUp;
+
+    /**
+     * @var string
+     */
+    public $tenantId;
+
+    /**
+     * @var []LoadbalancerListener
```

Comment:
put square bracket at the end`LoadBalancerListener[]`. Refer to https://phpdoc.org/docs/latest/guides/types.html#arrays